faster_whisper/transcribe.py

```diff
@@ -114,6 +114,7 @@ def __init__(
         self,
         model,
     ):
+        self.logger = get_logger()
```

Comment:
`self.model` already has a logger, so I'd rather we use the same logger instead of having duplicate loggers in both the pipeline and the model